shared/src/test/diff/codegen/Terms.mls

```diff
@@ -96,7 +96,7 @@ T { x = 0 }
 :js
 A { a = 0 }
 //│ // Query 1
-//│ res = new A({ a: 0 });
+//│ res = ((x) => new A(x))({ a: 0 });
```

Comment:
For performance and readability reasons, we should avoid doing this in applied constructor expressions, as in here!

Reply:
Fixed!